docker-compose-arlas-server.yaml

```diff
@@ -0,0 +1,39 @@
+version: '3'
+services:
+
+#################
+# ARLAS SERVER
+#################
+  arlas-server:
+    image: gisaia/arlas-server:${ARLAS_SERVER_VERSION:-latest}
+    container_name: arlas-server
+    restart: always
+    depends_on:
+      elasticsearch:
+        condition: service_healthy
+    expose:
+      - "9999"
+    ports:
```

Comment:
Is it necessary to keep this exposition ? we can interact with localhost:81/server no ?

Reply:
ok

Reply:
ok